client/k8s.go

```diff
@@ -224,6 +227,10 @@ func (k K8sClient) GetCVR(name string) (*cstorv1.CStorVolumeReplicaList, error)
 	if err != nil {
 		return nil, errors.Wrapf(err, "error while getting cStor Volume Replica for volume %s", name)
 	}
+	if len(CStorVolumeReplicas.Items) == 0 {
+		// TODO: This came during rebase, this shouldn't be required
+		klog.Errorf("Error while getting cStor Volume Replica for  %s , no replicas found", name)
```

Comment:
- Do we want to show this to user as a log? Maybe a fmt.Println would have been good.

Reply:
I'll move it to show info to `fmt.Printf` or via klog(but without the file info), the error should be printed to `stderr`, a simple `fmt.Println` would print it to `stdout`.